src/libs/OptionsListUtils.ts

```diff
@@ -1867,7 +1867,7 @@ function getOptions(
         allPersonalDetailsOptions = lodashOrderBy(allPersonalDetailsOptions, [(personalDetail) => personalDetail.text?.toLowerCase()], 'asc');
     }
 
-    const optionsToExclude: Option[] = [{login: CONST.EMAIL.NOTIFICATIONS}];
+    const optionsToExclude: Option[] = [];
```

Comment:
Coming from here https://github.com/Expensify/App/issues/49673, we decide to exclude `CONST.EMAIL.NOTIFICATIONS` again and it handled here https://github.com/Expensify/App/pull/50937